src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -547,22 +547,21 @@
      * is an optional identifier.
      *
      * <p>For example:</p>
-     *
      * <pre>
-     * extends java.util.LinkedList
+     * public class Test extends ArrayList {}
```

Comment:
lets have right curly braces on the next line

Reply:
ok

Reply:
@timurt done